.env.example

```diff
@@ -1 +1,5 @@
 VITE_JWT_PUBLIC_KEY="multiline-private-key-contents"
+VITE_GITHUB_OAUTH_APP_CLIENT_ID=
+VITE_GITHUB_OAUTH_APP_REDIRECT_URI=
+VITE_APP_BASE_URL=http://localhost:5173
+VITE_BACKEND_BASE_URL=http://localhost:8000
```

Comment:
I think it's a good time for me to setup a doppler instance for our project so when new secrets need to be added, the other programmer(s) have access to them without having to ask each other for them. Let me know your thoughts.